test/e2e/capi_test.go

```diff
@@ -223,33 +221,12 @@ var _ = Describe("Running the Cluster API E2E tests", func() {
 				Expect(os.Unsetenv("WINDOWS_WORKER_MACHINE_COUNT")).To(Succeed())
 
 				Expect(os.Setenv("K8S_FEATURE_GATES", "WindowsHostProcessContainers=true")).To(Succeed())
-
-				e2eConfigNoAddons = &clusterctl.E2EConfig{}
-				e2eConfig.ManagementClusterName = e2eConfigNoAddons.ManagementClusterName
-				e2eConfigNoAddons.Variables = e2eConfig.Variables
-				e2eConfigNoAddons.Intervals = e2eConfig.Intervals
-
-				providers := []clusterctl.ProviderConfig{}
-				for _, provider := range e2eConfig.Providers {
-					if provider.Type != "AddonProvider" {
-						providers = append(providers, provider)
-					}
-				}
-				e2eConfigNoAddons.Providers = providers
-
-				images := []clusterctl.ContainerImage{}
-				for _, image := range e2eConfig.Images {
-					if !strings.Contains(image.Name, "cluster-api-helm") {
-						images = append(images, image)
-					}
-				}
-				e2eConfigNoAddons.Images = images
```

Comment:
Does the `cluster-api-helm` image get auto included with this PR's change?

Reply:
Yes, before we were explicitly removing the cluster-api-helm image since the old version of CAPI we were was from before add-on providers were added to CAPI.